core/src/main/java/org/geysermc/geyser/text/MinecraftLocale.java

```diff
@@ -44,16 +45,15 @@ public class MinecraftLocale {
 
     public static final Map<String, Map<String, String>> LOCALE_MAPPINGS = new HashMap<>();
 
-    static {
-        // Create the locales folder
-        File localesFolder = GeyserImpl.getInstance().getBootstrap().getConfigFolder().resolve("locales").toFile();
-        //noinspection ResultOfMethodCallIgnored
-        localesFolder.mkdir();
+    private static final Path LOCALE_FOLDER = GeyserImpl.getInstance().getBootstrap().getConfigFolder().resolve("locales");
 
-        // FIXME TEMPORARY
+    static {
         try {
-            Files.delete(localesFolder.toPath().resolve("en_us.hash"));
-        } catch (IOException ignored) {
+            // Create the locales folder
+            Files.createDirectories(LOCALE_FOLDER);
+            Files.createDirectories(LOCALE_FOLDER.resolve("overrides"));
+        } catch (IOException exception) {
+            throw new RuntimeException("Unable to create locale folders! " + exception.getMessage());
```

Comment:
Do we want to stop Geyser from starting?

Reply:
I would, yes. If something as simple as this fails - it wont fail if the directories already exist, that's handled by `createDirectories´ - then someone is likely also running into other issues.